data/theme/gnome-shell-sass/_endless.scss

```diff
@@ -1065,3 +1065,36 @@ $success_time_payg_color: #02b842;
   padding-right: 12px;
   margin-top: 6px;
 }
+
+.hack-tooltip-icon {
+  margin-right: 1em;
+}
+
+.hack-tooltip-title {
+  text-align: center;
+  font-weight: bold;
+  font-size: 12pt;
+}
+
+.hack-tooltip-desc {
+  text-align: left;
+  font-size: 11pt;
+  padding: 10px 5px;
+}
+
+.hack-tooltip-arrow {
+  -arrow-background-color: white;
+}
+
+.hack-tooltip {
+  background-color: white;
+  color: black;
+  border-radius: 12px;
+  box-shadow: 0 4px 4px 4px rgba(0, 0, 0, .2);
```

Comment:
i cannot see the left shadow for some reason.

Reply:
Yes, it looks like the shadow is cropped or something. To avoid this I've moved the shadow to the right.